aoc/src/main/scala/Day01.scala

```diff
@@ -0,0 +1,63 @@
+// --- Day 1: Not Quite Lisp ---
+//
+// Santa is trying to deliver presents in a large apartment building, but he can't find the right floor - the directions he got are a little confusing. He starts on the ground floor (floor 0) and then follows the instructions one character at a time.
+//
+// An opening parenthesis, (, means he should go up one floor, and a closing parenthesis, ), means he should go down one floor.
+//
+// The apartment building is very tall, and the basement is very deep; he will never find the top or bottom floors.
+//
+// For example:
+//
+// (()) and ()() both result in floor 0.
+// ((( and (()(()( both result in floor 3.
+// ))((((( also results in floor 3.
+// ()) and ))( both result in floor -1 (the first basement level).
+// ))) and )())()) both result in floor -3.
+//  To what floor do the instructions take Santa?
+//
+// Your puzzle answer was 138.
+
+import scala.io.*
+
+object Day01 extends App {
+
+  val input =
+  Source
+    .fromFile("aoc/src/resources/inputDay01.txt")
```

Comment:
note : `fromResource` takes filenames relative to the "aoc/src/main/resources/" directory - better than `fromFile`

Reply:
Work in progress: Had to change it to `.fromResource("./inputDay01.txt")`